integrated_channels/blackboard/admin/__init__.py

```diff
@@ -1,15 +1,29 @@
 """
 Admin integration for configuring Blackboard app to communicate with Blackboard systems.
 """
+from config_models.admin import ConfigurationModelAdmin
 
-from six.moves.urllib.parse import urljoin
-
-from django.conf import settings
 from django.contrib import admin
 from django.utils.html import format_html
 
-from enterprise.utils import get_configuration_value
-from integrated_channels.blackboard.models import BlackboardEnterpriseCustomerConfiguration
+from integrated_channels.blackboard.models import (
+    BlackboardEnterpriseCustomerConfiguration,
+    BlackboardGlobalConfiguration,
+)
+
+
+@admin.register(BlackboardGlobalConfiguration)
+class SAPSuccessFactorsGlobalConfigurationAdmin(ConfigurationModelAdmin):
```

Comment:
copy/paste typo with `SAPSuccessFactorsGlobalConfigurationAdmin`?

Reply:
at least I got the doc string 🤦